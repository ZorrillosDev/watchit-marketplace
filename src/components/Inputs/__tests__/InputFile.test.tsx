import React from 'react'
import { shallow } from 'enzyme'
import {
  InputFileButton,
  InputFileElement,
  InputFileIcon,
  InputFileLabel,
  InputFileLabelText
} from '@components/Inputs/InputFile'

/* eslint-disable  @typescript-eslint/consistent-type-assertions */

describe('<InputFile />', () => {
  it('should render input file element', () => {
    const component = shallow(
      <InputFileElement />
    )

    expect(component).toMatchSnapshot()
  })

  it('should render input file icon', () => {
    const component = shallow(
      <InputFileIcon />
    )

    expect(component).toMatchSnapshot()
  })

  it('should render input file label text', () => {
    const component = shallow(
      <InputFileLabelText />
    )

    expect(component).toMatchSnapshot()
  })

  it('should render input file label', () => {
    const component = shallow(
      <InputFileLabel isFull filled />
    )

    expect(component).toMatchSnapshot()
  })

  it('should render input file button', () => {
    const component = shallow(
      <InputFileButton href='www.com' />
    )

    expect(component).toMatchSnapshot()
  })
})
