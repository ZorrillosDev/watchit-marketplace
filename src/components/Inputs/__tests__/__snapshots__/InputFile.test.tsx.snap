// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputFile /> should render input file button 1`] = `ShallowWrapper {}`;

exports[`<InputFile /> should render input file element 1`] = `ShallowWrapper {}`;

exports[`<InputFile /> should render input file icon 1`] = `ShallowWrapper {}`;

exports[`<InputFile /> should render input file label 1`] = `ShallowWrapper {}`;

exports[`<InputFile /> should render input file label text 1`] = `ShallowWrapper {}`;
